qcodes/instrument_drivers/american_magnetics/AMI430_visa.py

```diff
@@ -561,31 +552,35 @@ def ask_raw(self, cmd: str) -> str:
         return result
 
 
-class AMI430_3D(Instrument):
+class AMI430(AMIModel430):
+    pass
+
+
+class AMIModel4303D(Instrument):
```

Comment:
perhaps we can allow an underscore before 3D for readability?
```suggestion
class AMIModel430_3D(Instrument):
```

Reply:
Nah, I think we want to stick to conformant names and pep8 does not endorse  `_` in class names

Reply:
ok, fair!